### DISCUSSION TITLE
Change iOS icon

### DISCUSSION THREAD

**Discussion by rie39414** [ente-io/ente#1028] [Enhancements]
I think the iOS app icon is ugly (black text "auth" on a white background) as well as confusing with the thousands that are in the app store.  

Here I made a horrible example using the screenshoot available on the ente auth's github (https://github.com/ente-io/auth/blob/main/screenshots/screenshots.png)

I would prefer any image, other apps allow you to change the icon in-app, for example Raivo OTP allows you to choose as an icon a kind of very cute tropical bird, which covers the entire space of the icon, which is a bit the point of design in iOS.

Here's a quick comparison between the actual icon and a my quick hypothesis

![ente iOS icon](https://github.com/ente-io/auth/assets/145783398/62a8dc6e-9f8e-44c3-86bf-2ecf5abac833)
![ente auth hypothesis](https://github.com/ente-io/auth/assets/145783398/a632d25f-08d9-47a2-ad5f-a8bddaba5fc1)

**Comment by vishnukvmd** [comment]
Hey, thanks for sharing this! Will keep the issue open until we switch our current icon 🙂

**Comment by ua741** [comment]
We now have a new icon and landing page for Ente Auth https://ente.io/auth :)

**Comment by Adafede** [comment]
With ios 18, will an adataptive icon be released soon? 😊 

EDIT: just saw https://github.com/ente-io/ente/pull/3535, great!
